OS/DongChyeon/2025-10-17-DongChyeon-프로세스_스레드_그리고_코루틴.md

```
@@ -0,0 +1,108 @@
+topic: 프로세스, 스레드 그리고 코루틴
+---
+
+### 1. 프로세스(Process)란?
+
+> 컴퓨터에서 연속적으로 실행되고 있는 프로그램
+
+![process.png](assets/process.png)
+
+- 즉, 프로세스는 메모리에 로드되어 CPU가 실행 중인 프로그램 단위
+- 각 프로세스는 독립된 메모리 공간(코드, 데이터, 힙, 스택)을 가짐
+    - 코드
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)